src/nsolid.h

```diff
@@ -961,7 +962,45 @@ class NODE_EXTERN Snapshot {
                           Cb&& cb,
                           Data&&... data);
 
+  /**
+   * @brief activates the heap profiler tracking objects from a specific JS
+   * thread.
+   *
+   * @param envinst SharedEnvInst of thread to take the snapshot from.
+   * @param redacted to get redacted strings in the heap snapshot.
+   * @param data variable number of arguments to be propagated to the callback.
```

Comment:
Which cb are we propagating to?

Reply:
Fixed in the latest revision! With the timer support!